assets/js/mobile-app-banner.js

```diff
@@ -1,28 +1,44 @@
-const isAlertsPage = () => {
-  const url = window.location.href;
-
-  return /\/alerts\/[\w.-]+/.test(url);
+const isNotAndroid = () => {
+  return !/Android/.test(navigator.userAgent);
 };
 
-const isIPhone = () => {
-  return /iPhone/.test(navigator.userAgent);
+const isNotCommuterRail = url => {
```

Comment:
I think we also need to filter out ferry pages. The banner is showing up, for instance, on http://localhost:4001/schedules/Boat-F4/alerts

<img width="1314" alt="Screenshot 2024-10-25 at 2 42 25 PM" src="https://github.com/user-attachments/assets/b55b5c9d-0ab6-45aa-ba1b-4ea1fa3e1379">